zigzag/zigzag_test_log_field.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+
+# ======================================================================================================================
+# Imports
+# ======================================================================================================================
+
+from __future__ import absolute_import
+
+
+class ZigZagTestLogField(object):
+
+    def __init__(self, **kwargs):
+
+        self._data = kwargs
+
+    def __setattr__(self, key, value):
+        self._data[key] = value
+
+    def __getattr__(self, item):
```

Comment:
No docstrings.

Reply:
Also, you're using `key` for setting the attribute, but `item` for getting.

Reply:
fixed